http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -310,6 +310,9 @@ private[leonardo] object LeoProfile extends MySQLProfile {
           AppControlledResourceStatus.stringToObject
             .getOrElse(s, throw ColumnDecodingException(s"invalid app controlled resource status ${s}"))
       )
+
+    implicit val instantSetParameter: SetParameter[Instant] =
```

Comment:
previously I just embedded the conversion in the query..but since we added the `destroyedDate` filter, which is also `Instant`. I think it's worth adding this implicit now so that we don't have to worry about the conversion everywhere